src/hooks/useConnectApp.js

```diff
@@ -1,10 +1,11 @@
+import UserService from '../services/User.service'
 export const useConnectAppMixin = {
   methods: {
     async connectApp () {
-      this.link = true
-      this.$emit('linked', true)
+      window.open(process.env.VUE_APP_ALPACA_INTEGRATION_URL, '_self')
     },
-    async removeApp () {
+    async removeApp (userId, store) {
```

Comment:
If the method removeApp is intended to be used for all apps (not just alpaca) in the future, maybe it would be best to parameterize which app is to be removed.

Reply:
Was this comment addressed? @tungleu @rileykc98

Reply:
yes